code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -148,3 +148,76 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	energy_release = 29000
 	requirements = list(/datum/gas/antinoblium = 5, /datum/gas/zauker = 20, /datum/gas/bz = 7.5)
 	products = list(/obj/item/stack/sheet/mineral/zaukerite = 2)
+
+/datum/gas_recipe/hypertorus_fusion_reactor
+	machine_type = "Hypertorus_Fusion_Reactor"
+	//Normal id for recipe
+	id = ""
+	//The gases that are going to be used as fuel ("GasX + GasY fuel")
+	name = ""
+	//Multiplier for the minimum heat output of the HFR (min 0.01)
+	min_temp = 0
+	//Multiplier for the maximum heat output of the HFR (min 0.01)
+	max_temp = 0
```

Comment:
These are really really misleading

Reply:
Yeah the vars are, I could use new vars on the datum to accomplish the same thing?